Makefile

```diff
@@ -8,6 +8,10 @@ all: build
 build-cli:
 	make -C bin/spice
 
+.PHONY: build-cli-dev
+build-cli-dev:
+	export DEV=true; make -C bin/spice
```

Comment:
Doesn't this also need `SPICED_NON_DEFAULT_FEATURES="models"`?

Reply:
that is only for cli build